mbld/main.myr

```diff
@@ -75,15 +75,23 @@ const main = {args : byte[:][:]
 		std.setenv(e, v)
 	;;
 
+	std.put("cmd.args: {}\n", cmd.args)
 	b = mkbuild(tags)
 	if targname.len != 0
 		if cmd.args.len == 0
 			std.fatal("nothing to compile\n")
 		;;
+
+		if runsrc
```

Comment:
Better: make '-R' take an argument (just ad a '.arg' field in the description on line 28, and save it in srcs on line 60). I'd just get rid of the 'runsrc' and check if the arg is non-empty in that case.

Then, the args passed to the runner here are just cmd.args.

The current way of doing it would be kind of confusing -- if I did:

     mbld foo.myt -R bar.myr

then it would run foo.myr and pass it bar.myr

Reply:
Isn't that the argument specified by `.arg` can only be a single string? 
Do you mean it should be tokenized before saving it in `srcs`?

Reply:
srcs = [runsrc][:]

The idea is that `-R` gets a proper arg, and we only support running a single file.

IThe alternative approach I was thinking of was that ,`cmd` should probably get a 'lastparsed` field, which records the position of the `--` argument, so we can just use `args = cmd.args[cmd.lastparsed:]`.

Reply:
Updated.